app/forms/candidate_interface/other_qualification_type_form.rb

```diff
@@ -58,5 +70,16 @@ def intermediate_state
     def qualification_type_changed?
       id && ApplicationQualification.find(id)&.qualification_type != qualification_type
     end
+
+    def prepare(options)
+      options.select { |_, value| value.present? }
```

Comment:
Does this line do anything? (it doesn't mutate `options`)

Reply:
Yep this is an error on my part - have fixed